util/keyfile/src/bin/convert_to_pubfile.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Processees pubkeys to file format understood by ingest add_user.
+//!
+//! This tool can be followed up by a call to add_user, e.g. with
+//! `cargo run fog-ingest-client -- --uri fog-ingest://fog-ingest.NETWORKNAME.mobilecoin.com:443 add-users --keys-path ./fog_keys`
+
+use mc_account_keys::PublicAddress;
+use mc_api::external;
+use mc_util_keyfile::write_pubfile;
+use protobuf::parse_from_bytes;
+use std::{convert::TryFrom, fs, path::PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Config {
+    /// Path to input pubfile dir
+    ///
+    /// Note: Pubfile format is expected to be a the hex-encoded protobuf bytes.
+    #[structopt(long)]
+    pub pubfile_in_dir: PathBuf,
+
+    /// Path to output pubfile dir
+    #[structopt(long)]
+    pub pubfile_out_dir: PathBuf,
+}
+
+fn read_to_pubkeys(
+    pubfile_in_dir: PathBuf,
+) -> Result<Vec<(PublicAddress, PathBuf)>, std::io::Error> {
+    let mut pub_addrs: Vec<(PublicAddress, PathBuf)> = Vec::new();
+    for entry in fs::read_dir(pubfile_in_dir)? {
+        let entry_path = entry?.path();
+        let pub_str = fs::read_to_string(entry_path.clone())?;
+        let pub_string: String = pub_str.split_whitespace().collect();
+        let data = hex::decode(pub_string).expect("Could not hex decode public address string");
+        let pub_addr_proto: external::PublicAddress =
+            parse_from_bytes(&data).expect("Could not parse public address from bytes");
+        let pub_addr = PublicAddress::try_from(&pub_addr_proto)
```

Comment:
`PublicAddress` implements Prost, so why go through protobuf->prost and not mc_util_serial::decode instead?